### PR TITLE
Test PR for update flow

### PR DESCRIPTION
**Pull request by eduard-tita**
Test PR for update flow

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>org.apache.struts</groupId>
       <artifactId>struts2-core</artifactId>
-      <version>2.5.14</version>
+      <version>2.3.15</version>
```
Comment (eduard-tita) [inline pom.xml]: Original lien comment

File: pom.xml [hunk 2]
```diff
@@ -35,14 +35,18 @@
         <version>${junit.version}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.apache.struts</groupId>
+        <artifactId>struts2-core</artifactId>
+        <version>2.5.22</version>
```
Comment (eduard-tita) [inline pom.xml]: New line comment

File: pom.xml [hunk 3]
```diff
@@ -27,22 +28,27 @@
     </repository>
   </repositories>
 
-  <dependencyManagement>
+  <dependencyManagement>           
     <dependencies>
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>
```
Comment (eduard-tita) [inline pom.xml]: DELETE ME!

Comment (louisrdev) [inline pom.xml]: NEW COMMENT

Comment (louisrdev) [inline pom.xml]: NEW COMMENT AFTER DELETE AND RECREATE

Comment (eduard-tita) [reply pom.xml]: Some more comments after the comment to be deleted.

**Pull request closed without merging**